packages/backend/src/apps/aisay/auth/is-still-verified.ts

```diff
@@ -0,0 +1,10 @@
+import { IGlobalVariable } from '@plumber/types'
+
+import verifyCredentials from '../../custom-api/auth/verify-credentials'
+
+const isStillVerified = async ($: IGlobalVariable) => {
+  await verifyCredentials($)
+  return true
+}
```

Comment:
this one should import from aisay instead oops i was figuring out why connection keeps displaying as `unnamed` when i was testing

Reply:
can double check with aisay if they have a simple API call to verify if it works? if don't have nvm also